marameref/appwrite
# Update login-with-one-time-credential.md

## What does this PR do?

This PR contains detailed step by step tutorial documentation on How to log in (to the appwrite application)  through a one-time credential 

## Test Plan

- Start your app and navigate to the login page or route.

- Click the magic link generated in (The process of generating the magic link is well explained in the documentation)

- Verify that the user is successfully authenticated and redirected to the appropriate page.

## Related PRs and Issues

- Documentation: How to log in through a one-time credential #6447

## Checklist
